build-run.js

```diff
@@ -0,0 +1,58 @@
+#! /usr/bin/env node
+
+const yargs = require('yargs/yargs');
+const { hideBin } = require('yargs/helpers');
+
+const { spawn } = require('child_process');
+
+const args = yargs(hideBin(process.argv))
+  .option('env', {
+    alias: 'e',
+    type: 'string',
+    default: 'sit',
+    describe: 'nextcapital environment to use (sit or development)'
+  })
+  .option('solution', {
+    alias: 's',
+    type: 'string',
+    default: 'nextcapital',
+    describe: 'solution to use'
+  })
```

Comment:
Just a thought, do we want anyone with access to this demo to be able to run Next Gen with any of our possible solutions? Like would it be a problem if a prudential dev checked out the TA or newport experience?

Reply:
It's a longstanding issue: We may solve this by removing the `solutionId` argument and making a request to the server to determine solution. This would need to be another major version.

In any case, partners can guess URLs to see other solutions today, so I don't think this is a big deal.